packages/@aws-cdk/aws-logs/test/integ.save-logs-insights-query-definition.expected.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Resources": {
+    "QueryDefinition4190BC36": {
+      "Type": "AWS::Logs::QueryDefinition",
+      "Properties": {
+        "Name": "MyQuery",
+        "QueryString": "fields @timestamp, @message | sort @timestamp desc | limit 20"
+      }
+    }
+  }
+}
```

Comment:
could we get a blank line at the end of the file?

Reply:
Fixed.